sdk/js-connect/README.md

```diff
@@ -0,0 +1,338 @@
+  emitter: toNative('Ethereum', emitterAddress).toUniversalAddress(),
+  sequence: '0x1234...',
+});
+
+const dstTxIds = await xfer.completeTransfer(dst.signer)
+```
+
+## Concepts
+
+Understanding several higher level concepts of the SDK will help in using it effectively.
+
+### Platforms
+
+Every chain is its own special snowflake but many of them share similar functionality.  The `Platform` modules provide a consistent interface for interacting with the chains that share a platform.
+
+Each platform can be installed separately so that dependencies can stay as minimal as possible.
+
+
+### Chain Context
+
+The `Wormhole` class provides a `getChain` method that returns a `ChainContext` object for a given chain.  This object provides access to the chain specific methods and utilities.  Much of the functionality in the `ChainContext` is provided by the `Platform` methods but the specific chain may have overriden methods. 
+
+The ChainContext object is also responsible for holding a cached rpc client and protocol clients. 
+
+```ts
+// Get the chain context for the source and destination chains
+// This is useful to grab direct clients for the protocols 
+const srcChain = wh.getChain(senderAddress.chain);
+const dstChain = wh.getChain(receiverAddress.chain);
+
+
+srcChain.parseAddress("0xdeadbeef...") // => NativeAddress<'Evm'>
+await srcChain.getTokenBridge() // => TokenBridge<'Evm'>
+srcChain.getRpcClient() // => RpcClient<'Evm'>
+```
+
+### Protocols
+
+While Wormhole itself is a Generic Message Passing protocol, a number of protocols have been built on top of it to provide specific functionality. 
+
+#### Token Bridge
+
+The most familiar protocol built on Wormhole is the Token Bridge.
+
+Every chain has a `TokenBridge` protocol client that provides a consistent interface for interacting with the Token Bridge.  This includes methods to generate the transactions required to transfer tokens, as well as methods to generate and redeem attestations. 
+
+Using the `WormholeTransfer` abstractions is the recommended way to interact with these `Protocols` but it is possible to use them directly
+
+```ts
+import {signSendWait} from '@wormhole-foundation/wormhole-sdk';
+
+import "@wormhole-foundation/wormhole-sdk-evm-core"
+import "@wormhole-foundation/wormhole-sdk-evm-tokenbridge"
+
+// ...
+
+const tb = await srcChain.getTokenBridge() // => TokenBridge<'Evm'>
+
+const token = '0xdeadbeef...';
+const txGenerator = tb.createAttestation(token) // => AsyncGenerator<UnsignedTransaction, ...> 
+const txids = await signSendWait(srcChain, txGenerator, src.signer) // => TxHash[]
+
+```
+
+Supported protocols are defined in the [definitions module](https://github.com/wormhole-foundation/connect-sdk/tree/develop/core/definitions/src/protocols)
+
+### Signers
+
+In order to sign transactions, an object that fulfils the `Signer` interface is required.  This is a simple interface that can be implemented by wrapping a web wallet or other signing mechanism.  
```

Comment:
Down the line, it would be super cool to see a tutorial on hooking this up with [aws kms based ethereum signers](https://aws.amazon.com/blogs/database/part1-use-aws-kms-to-securely-manage-ethereum-accounts/).